### ISSUE TITLE
CMake version for build

### ISSUE THREAD

**Issue by AndrewHerzing** [tomopy/tomopy#473]
**Describe the problem**
I am attempting to build tomopy on a Linux ppc64le cluster that has two versions of CMake installed (2.8.12.2 and 3.13.4).  The build is failing because it attempts to use the older version which is not suitable.  I am not sure how to force use of the newer version of CMake.  Is there an environment variable to set or does the setup.py file need to be edited?

**To Reproduce (if applicable)**

```python
cd /home/user/tomopysourcedirectory
python setup.py install 
```

```
CMake Error at CMakeLists.txt:2 (cmake_minimum_required):
  CMake 3.9 or higher is required.  You are running version 2.8.12.2

```

**Comment by carterbox** [comment]
Hi @AndrewHerzing,

TomoPy currently [requires Intel's MKL](https://github.com/conda-forge/tomopy-feedstock/pull/19#issuecomment-587909203). This probably is a bigger obstacle than getting scikit-build to find the CMake binary that you want.

For the later, please see the [scikit-build docs.](https://scikit-build.readthedocs.io/en/latest/usage.html#usage-scikit-build-options) There is a parameter you can pass if the CMake you want is not the first one on your path.

**Comment by AndrewHerzing** [comment]
Thanks.  I was able to get around the CMake problem by installing cmake with conda.  Now I am up against the MKL problem, as you predicted.  I guess there is no hope for now.

**Comment by jrmadsen** [comment]
@AndrewHerzing you just need to prefix your PATH environment variable with the path to the latest cmake: https://en.m.wikipedia.org/wiki/PATH_(variable)

If you are on a cluster, there is likely a module that will do this: `module avail cmake` to see options and `module load cmake/<VERSION>`

**Comment by AndrewHerzing** [comment]
Have your or anyone else tried to build without MKL using OpenBLAS or an alternative?

**Comment by jrmadsen** [comment]
@AndrewHerzing It won't work, CMake explicitly looks for MKL libraries.

@dgursoy @carterbox People really do seem to hate that MKL dependency... if only gridrec wasn't the most commonly used reconstruction method...

**Comment by carterbox** [comment]
@AndrewHerzing, please look at the open pull requests (#458). In theory, some people at Oak Ridge are working on implementing a POWER8 compatible gridrec.

@jrmadsen, yeah, it seems we forgot that MKL isn't available for all architectures and some people don't like it's license. :man_shrugging: 

Also, this summer, I have an intern working on breaking the TomoPy compiled library into sublibraries (#452), so at least people will be able to use all the parts of the library that don't depend on MKL.

**Comment by carterbox** [comment]
Resolved by #512
